models/issue_stopwatch.go

```diff
@@ -160,3 +171,31 @@ func SecToTime(duration int64) string {
 
 	return hrs
 }
+
+// APIFormat convert Stopwatch type to api.StopWatch type
+func (sw *Stopwatch) APIFormat() (api.StopWatch, error) {
+	issue, err := getIssueByID(x, sw.IssueID)
+	if err != nil {
+		return api.StopWatch{}, err
+	}
+	return api.StopWatch{
+		Created:    sw.CreatedUnix.AsTime(),
+		IssueIndex: issue.Index,
+	}, nil
+}
+
+// Stopwatches is a List ful of Stopwatch
+type Stopwatches []Stopwatch
+
+// APIFormat convert Stopwatches type to api.StopWatches type
+func (sws Stopwatches) APIFormat() (api.StopWatches, error) {
+	var result api.StopWatches
```

Comment:
Hmm... Probably best to just create this to the correct size and then cast.

```suggestion
	result := api.StopWatches(make([]api.StopWatch, 0, len(sws)))
```